index.html

```diff
@@ -4158,6 +4182,15 @@ <h3>
 The MIME type of the returned <code>content-stream</code> SHOULD be expressed
 using this property if dereferencing is successful.
                 </dd>
+
+                <dt>
+deactivated
```

Comment:
Why does this show up twice in the spec? With different normative requirements?

Reply:
It currently shows up twice in this PR because just like "contentType" and "error", it's included both in the `did-resolution-metadata` and in the `did-url-dereferencing-metadata`.

But as I noted in my previous comment, that's a mistake, since it should only show up once in the `did-document-metadata`.